packages/backend-function/src/factory.test.ts

```diff
@@ -337,5 +337,15 @@ void describe('AmplifyFunctionFactory', () => {
         new Error('runtime must be one of the following: 16, 18, 20')
       );
     });
+
+    void it('throws when the oldest maintained Node LTS reaches end of life', () => {
+      // A month before the date when the oldest Node LTS maintenance ends according to https://github.com/nodejs/release#release-schedule.
+      // Once this test fails, update endDate to a month before the end date of the next Node LTS version.
+      // After updating the date, next would be updating the default function runtime in factory.ts.
+      const endDate = new Date('2025-03-30');
+      const currentDate = new Date();
+
+      assert.equal(endDate > currentDate, true);
```

Comment:
nit: 
```suggestion
      assert.ok(endDate > currentDate);
```

Reply:
Oh I wasn't aware of this shorthand for what I was doing